app/src/common/css/variables/newColors.scss

```diff
@@ -54,3 +54,4 @@ $COLOR--tag-value-text: #394db6;
 $COLOR--tag-value-background: #ced8fc;
 $COLOR--tag-key-text: #6f4599;
 $COLOR--tag-key-background: #dac3e6;
+$COLOR--grey-90: #e5e5e5;
```

Comment:
Does it agreed with UX? 
Look like it's not consistent with existing design system also I cannot find this color in figma